### DISCUSSION TITLE
Zoom and Deleting a Log

### DISCUSSION THREAD

**Discussion by jandreaucodes** [mockoon/mockoon#1482] [General]
Two questions:

1) Why is the zoom limited? After a certain point, you can't zoom in on the app anymore. This makes it difficult for people to see your app on screensharing when you have a much higher resolution screen than the viewers do.
2) It doesn't seem to be possible to delete a single log, just clearing all of the logs. Can someone confirm, so I can submit a feature request for single log deleting?

**Comment by 255kb** [comment]
> Two questions:
> 
> 1. Why is the zoom limited? After a certain point, you can't zoom in on the app anymore. This makes it difficult for people to see your app on screensharing when you have a much higher resolution screen than the viewers do.

We can definitely increase the zoom limit. My concern was to avoid breaking the UI on small screens and high zoom, but I tested and it is already a bit "broken" (content is overflowing the viewport). I guess this is another issue that shouldn't prevent us to allow for a higher zoom level. 

> 2. It doesn't seem to be possible to delete a single log, just clearing all of the logs. Can someone confirm, so I can submit a feature request for single log deleting?

Indeed there is no such option. Now that there is a context menu on each entry it would be quite simple to add such feature. Also, please note that logs are not persisted on the hard drive as a whole or individually. 

You can open issues for both feature requests. If you would like to contribute and implement the features, that would be great too!
Thanks
